coordinator/src/services/validation/bundle-validator.ts

```diff
@@ -165,7 +176,7 @@ function verifyMessageIsResource(message: unknown): message is fhir.Resource {
 
 function verifyResourceTypeIsBundle(resource: unknown): resource is fhir.Bundle {
   return verifyMessageIsResource(resource)
-        && resource.resourceType === "Bundle"
+    && resource.resourceType === "Bundle"
```

Comment:
verifying the resourceType is Bundle is presumably the easier check -- could you flip the conditions around the `&&` to make use of short-circuiting?